product_docs/docs/pgd/5/upgrades/tpa_overview.mdx

```diff
@@ -24,11 +24,11 @@ Major version upgrades of PGD require changes to the TPA `config.yml` file, whic
 
 When upgrading to PGD 5 from previous PGD major versions, you can use [`tpaexec reconfigure`](/tpa/latest/reference/tpaexec-reconfigure/). This command helps you make appropriate modifications to your deployment configuration. 
 
-The `reconfigure` command requires settings for architecture (only `PGD_Always_ON` and PGD Proxy routing to run. Remember to back up your deployment configuration before running, and use the `--describe` and `--output` options to preview the reconfiguration. 
+The `reconfigure` command requires settings for architecture-only `PGD_Always_ON` and PGD Proxy routing to run. Remember to back up your deployment configuration before running, and use the `--describe` and `--output` options to preview the reconfiguration. 
```

Comment:
@djw-m Please check this edit. There was an errant parentheses and I'm not sure that the edit is correct.

Reply:
```suggestion
The `reconfigure` command requires settings for architecture (the only supported setting is `PGD_Always_ON`) and PGD Proxy routing (`--pgd-proxy-routing <global|local>`) to run. Remember to back up your deployment configuration before running, and use the `--describe` and `--output` options to preview the reconfiguration. 
```